server/csharp/TicketHub/Authorization/QueryableExtensions.cs

```diff
@@ -79,4 +113,66 @@ private static Expression BuildCompoundExpression<T>(UCASTNode node, ParameterEx
             _ => throw new ArgumentException($"Unknown compound operator: {node.Op}"),
         };
     }
+
+    /// <summary>
```

Comment:
I like these comments! Would they show up in auto-generated docs if/when we spin this out into a library? 🤔

Reply:
Yep! That's why I put the extra effort in with the [XML docs comments](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags). Intellisense in VS Code picks up the `<summary>` tags, for example, which is really handy.

The formatting is mangled in the unordered list below, but this is generally what it looks like. :sweat_smile: 
![image](https://github.com/user-attachments/assets/a3a58d96-7a64-467f-ae90-b464bcd9787d)

Reply:
Here's a cleaned up version; I'll push up the XML changes shortly...
![image](https://github.com/user-attachments/assets/f6356d94-09fb-47ba-981f-ff0d5f43e9db)